tutorials/javaFxPart3.md

```diff
@@ -78,71 +80,14 @@ public void start(Stage stage) {
 }
 ```
 
-Verify that the `ScrollPane` scrolls as intended.
-
-## Iteration 2 – Adding Dialog Boxes
-
-In the mockup of the UI, notice that the dialog boxes are composed of two different controls (`ImageView` and `Label`) and reused multiple times. In situations like this, it is often beneficial to create our own custom control.
-
-Let’s create our custom control `DialogBox`:
+Verify that the `ScrollPane` scrolls as intended. Feel free to use the following long message to test your program:
 ```java
```

Comment:
remove `java`, as this is not a Java code block.

Reply:
Hi professor, I orignally put this inside a code block because of 1. easy copying of the entire paragraph and 2. taking up little space in the page. Could you advise other ways to achieve this? I did this because I tought making the dummy text take up space like normal text would be distracting.

Reply:
It can be a code block without marking it as a Java code block. More info https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Reply:
I see what you mean. Thank you prof, the new commit is made.